tests/compval

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import click
+import json
+from jsondiff import diff
+import pandas as pd
+import sys
+
+@click.command()
+@click.argument('json_path', nargs = 1 , required = True)
+def main(json_path):
+
+    #Get/set cv path
+    cv_path = './CMIP6_frequency.json'
```

Comment:
The cv path will be different for each vocabulary item, and it probably won't be called "CMIP6_frequency", and it won't be in the current directory.

We want to retrieve the vocabulary referenced in the schema. e.g.

https://github.com/aradhakrishnanGFDL/CatalogBuilder/blob/main/cats/gfdl_template.json#L22

But also the chunk_freq vocabulary:

https://github.com/aradhakrishnanGFDL/CatalogBuilder/blob/main/cats/gfdl_template.json#L50